### ISSUE TITLE
Looting enchantment not working as expected.

### ISSUE THREAD

**Issue by Vadewolf** [MarkusBordihn/BOs-Easy-Mob-Farm#35]
**Describe the bug**
the mob farms do not use looting 3 on swords

**To Reproduce**

**Expected behavior**
for every enchant to be used pls
**Screenshots**

**Debugging**
Before reporting a issue please check if there is anything relevant in the crash-reports, debug.log and latest.log file.

**Is the issue happening in combination with other mods? (If yes, please provide a full list)**

**Additional context**
basically the mob farms do not use looting 3 on swords even know they should use every enchant on sword
![2023-10-11_10 19 26](https://github.com/MarkusBordihn/BOs-Easy-Mob-Farm/assets/18257282/e46b9ba7-abab-4799-bdb9-8be76366b08a)

**Comment by MarkusBordihn** [comment]
Thanks for the report. Just to make sure what you understand as "do not use".

Currently the mod farm is increasing the luck with each "looting" level, which results in better and more drops in most cases.
But this will not help with a simple entity like a cow, so I assume that you expect that it multiply the loot drops as well like:

- 2 x for Looting I
- 3 x for Looting II
- 4 x for Looting III

Correct ? If yes, I will implement this with the next update, because it seems that the standard loot tables are not considering these at the moment.

**Comment by Vadewolf** [comment]
basically the mob farms are dropping 1 item per "kill" even with a fully maxed sword as shown above and yes pls cause getting 1 drop per kill when it should use all the enchants for player specfic kills it doesnt

**Comment by MarkusBordihn** [comment]
This should be fixed with 6.6.2 which adds additional rolls depending on the looting level.
But keep in mind that additional rolls doesn't mean automatically double or tripple item drops.
Most drops still based on luck, so if you have very bad luck you could get 0 items even with Looting III and 4 rolls.
